source/documentation/guides/govuk-paas-suitability.html.md

```diff
@@ -0,0 +1,48 @@
+| Feature  | Azure | Gov UK PaaS |
+| :---         | :---         | :---         |
+| Service Management Wrap Cost   | TBD | 10% Overhead    |
+| Need for DevOps/Infra Ops engineers    | Yes       | Dependent (e.g. need for ad-hoc basis for such as High availability configuration and similar examples)      |
+| Disaster Recovery and Load Balancing    | Yes, configurable       | Yes, but not configurable at a low level      |
+| Auto and horizontal scaling    | Yes, configurable       | Yes, configurable      |
+| Rapid deployment    | Yes       | Yes      |
+| Nature of Ecosystem    | IaaS and PaaS       | PaaS only      |
+| Compliance with DfE Standards    | Yes       | Yes      |
+| Public facing type apps/services    | Configurable       | Suited (configurational setup)     |
+| Lightweight Apps    | Acceptable       | Suited (Time to market)     |
+| Time-Bound Prototypying    | Acceptable       | Suited (Time to market)      |
+| OTS Platform Security    | Good, via security rules       | Good, built in the platform      |
+| Areas of Operation    | Full techn stack       | PaaS only      |
+| GitHub Integration    | TBD       | Configured by service      |
```

Comment:
Github is independent from the hosting platform.
Azure devops integrates well with Github. I think both entries could be "Configured by service"

Reply:
Corrected 'Prototyping' and 'technical'

Reply:
Updated the GitHub entry